.gitignore

```diff
@@ -75,3 +80,11 @@ ipch/
 ## Mac OSX
 #################
 .DS_Store
+
+#################
+## Clion (Jetbrains IDEs)
+#################
+# Project files
+.idea/*
+# CMake
+cmake-build-*/
```

Comment:
This change has also already been committed to `master` (commit 2d84c4182712432ae3402e3ccee771b5dc04e471) so it will be ignored when your PR will be merged.